### ISSUE TITLE
### <span aria-hidden="true">❌</span> Deploy Preview for *gonnaai* failed.

### ISSUE THREAD

**Issue by amethystani** [Gonna-AI/Frontend#3]
### <span aria-hidden="true">❌</span> Deploy Preview for *gonnaai* failed.

|  Name | Link |
|:-:|------------------------|
|<span aria-hidden="true">🔨</span> Latest commit | 501f8c70c4dc2497e87dd356678c0d3aca42c57b |
|<span aria-hidden="true">🔍</span> Latest deploy log | https://app.netlify.com/sites/gonnaai/deploys/676aef0bdf524300081ac7ff |

_Originally posted by @netlify[bot] in https://github.com/Gonna-AI/Frontend/issues/2#issuecomment-2561307154_
